lib/dwolla.ex

```
@@ -101,30 +103,31 @@ defmodule Dwolla do
   end
 
   def process_response_body(""), do: ""
+
   def process_response_body(body) do
-    case Poison.Parser.parse(body) do
-      {:ok, parsed_body} -> parsed_body
-      {:error, _}        -> {:invalid, body}
-    end
+    Poison.Parser.parse!(body, %{})
```

@cmschuetz why the choice to raise on failure to parse instead of returns a tuple?